spec/models/allowed_payment_method_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe AllowedPaymentMethod do
```

Comment:
We can DRY these duplicated reference data specs with `it_behaves_like`, see my example with the specs for reference data for `Adviser`